packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DerivApps from '../../public/images/deriv-apps.svg';
+import { WalletButton, WalletText } from '../Base';
+
+type TProps = {
+    isMaltaWallet?: boolean;
```

Comment:
```suggestion
       isMaltaWallet: NonNullable<ReturnType<typeof useActiveWalletAccount>['data']>['is_malta_wallet'];
```

Just a suggestion @sergei-deriv. At the end of the day its just a boolean but i think its better for  readability since its a prop and we have to go and find where is it coming from and which API is it coming from. So if we add this here, at one look we know its coming from `useActiveWalletAccount`. We dont need to waste time finding where its originating from. 
Feel free to ignore my comment 😸